backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/importer/general/ImportToDatabase.kt

```diff
@@ -29,25 +28,36 @@ object ImportToDatabase {
         ).replace("<br/>", "\n")
     }
 
+    /**
+     * Replaces notes for unspecified entries with empty strings.
+     *
+     * Such a note consists of the letters "nA" (in that order), potentially with dots like "n.A." and/or stuffed with whitespace (e.g. " n A. ").
+     *
+     * @return empty string if `text` is a "unspecified note", `text` otherwise
+     */
+    private fun replaceUnspecifiedNote(text: String): String {
+        return if (Regex("""\s*n\.?\s*A\.?\s*""").matches(text)) "" else text
```

Comment:
This regex should probably be stored as a variable in order to speedup the matching

Reply:
Done